spec/models/google_doc_fetcher_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe "GoogleDocFetcher" do
+  it "supports any valid URL" do
+    expect(GoogleDocFetcher.supports_url?("https://drive.google.com/open?id=1Ngds9Fr4vas8n0cA-uvZDOU7VMarMfGytbC9VLc0IYI")).to be_truthy
+    expect(GoogleDocFetcher.supports_url?("https://docs.google.com/document/d/1-waDJtPB8VGLyubb40-X951PF9R3jdhm6Kb4ga__S0E/edit")).to be_truthy
+  end
+
+  it "fetches a google doc export" do
+    http_setup_get_request_mock_success
+
+    params = {"url" => "https://docs.google.com/document/d/1-waDJtPB8VGLyubb40-X951PF9R3jdhm6Kb4ga__S0E/edit"}
+    fetcher = GoogleDocFetcher.new(params)
+    res = fetcher.fetch_content
+    expect(!res.empty?)
```

Comment:
As discussed in our meeting, I changed this to `expect(res).to be_falsely` but the test fails.  Apparently in ruby empty strings are truthy!  
```
  1) WebsiteFetcher fetches a bogus website and returns an empty string
     Failure/Error: expect(res).to be_falsey
       expected: falsey value
            got: ""
     # ./spec/models/website_fetcher_spec.rb:24:in `block (2 levels) in <top (required)>'
```
The interface here was intended to be that fetch_content performs the request, sanitizes the content, and returns a string.  Here is how the model is used
```
        content = fetcher.fetch_content
        if !content.empty?
          self.upload_file(id, team.id, content, file_number)
        end
```
Not sure how you'd like to proceed at this point, please advise.